pkg/policyengine/loadBalancer.go

```diff
@@ -30,233 +33,180 @@ const (
 	Static LBScheme = "static"
 )
 
-const Wildcard = "*"
-
+// LBPolicy is being used by the CRUD interface, and is now deprecated.
 type LBPolicy struct {
 	ServiceSrc  string
 	ServiceDst  string
 	Scheme      LBScheme
 	DefaultPeer string
 }
 
-type ServiceState struct {
+type serviceState struct {
+	scheme           LBScheme
+	impSources       []crds.ImportSource
 	totalConnections int
-	defaultPeer      string
 }
 
 type LoadBalancer struct {
-	ServiceMap      map[string][]string                 // Service to Peers
-	Scheme          map[string]map[string]LBScheme      // PolicyMap [serviceDst][serviceSrc]Policy
-	ServiceStateMap map[string]map[string]*ServiceState // State of policy Per destination and source
+	services map[string]*serviceState // Keeps a state for each imported service
```

Comment:
Why not change the key from `string` to `types.NamespacedName` and avoid the string concatenations?

Reply:
Done